src/seedu/addressbook/AddressBook.java

```diff
@@ -34,6 +35,30 @@
  * This class is used to maintain a list of person data which are saved
  * in a text file.
  **/
+/**
+ * @author Difeng
```

Comment:
errr! these are quite useless comments. I am assuming they crept in by mistake?
ensure that the comments are used to explain something about the code. if you need to let the reviewer know the intention, you can use GitHub comments (like this one) to communicate the same.